### DISCUSSION TITLE
Is there a way to disable circular reference checks in locals?

### DISCUSSION THREAD

**Discussion by syphernl** [terraform-linters/tflint#1564] [Q&A] [answered]
With Tflint 0.42.0 we now see quite a number of pipelines failing because tflint found circular references in locals created/used by upstream modules we use. 
These circular references however do not seem to be causing issues, is there a way to disable this particular check (for now) or do we have to downgrade to < 0.42.0?

**Comment by wata727** [comment, marked as answer]
This is a bug in v0.42.0. See https://github.com/terraform-linters/tflint/issues/1565.
For now, the only option is to downgrade, but a fix will be released soon.
